packages/api/src/config.js

```diff
@@ -114,11 +114,13 @@ export const products = {
       {
         name: "Storage",
         description: "Storage (minutes)",
-        price: 0.0035,
+        price: 0.0015,
         limit: 10_000,
       },
     ],
-    monthlyPrice: 100,
+    monthlyPrice: 1, // test value for staging
+    minimumSpend: true,
+    concurrent_viewers: 50_000,
```

Comment:
Can we use camel 🐫 for some consistent casing here? This is the only snake 🐍 
```suggestion
    concurrentViewers: 50_000,
```

Reply:
Finished the PR and came back here. Shouldn't this be enforced on the gate API?

Reply:
Right, will do camel!
Yeah that's enforced by the gate API - but this is just for the frontend display on the plan page. All of these products should end up in a `product` table in the future with billing v2, and both the frontend and the api will fetch from this field